internal/statesync/reactor.go

```diff
@@ -782,16 +777,16 @@ func (r *Reactor) handleParamsMessage(envelope p2p.Envelope) error {
 // It will handle errors and any possible panics gracefully. A caller can handle
 // any error returned by sending a PeerError on the respective channel.
 func (r *Reactor) handleMessage(chID p2p.ChannelID, envelope p2p.Envelope) (err error) {
-	defer func() {
-		if e := recover(); e != nil {
-			err = fmt.Errorf("panic in processing message: %v", e)
-			r.Logger.Error(
-				"recovering from processing message panic",
-				"err", err,
-				"stack", string(debug.Stack()),
-			)
-		}
-	}()
+	// defer func() {
```

Comment:
No longer needed?

Reply:
good catch. I commented out becasue it made it harder to see what was panic'ing (the mock) now things are good.